### ISSUE TITLE
Issues after 0.24.0 upgrade

### ISSUE THREAD

**Issue by Javieracost** [blinklabs-io/adder#259]
As the title says, upon upgrading to 0.24.0 I found that adder fails to connect to the node and shows these messages repeatedly:
```
{"timestamp":"2024-10-14T15:03:49Z","level":"INFO","msg":"reconnecting to cardano-node-mainnet:3001 due to error: protocol error: failed to create address from match.Address: string not all lowercase or all uppercase","component":"main","plugin":"input.chainsync"}
{"timestamp":"2024-10-14T15:03:49Z","level":"INFO","msg":"connected to node at cardano-node-mainnet:3001","component":"main","plugin":"input.chainsync"}
{"timestamp":"2024-10-14T15:03:49Z","level":"INFO","msg":"","component":"main","plugin":"output.log","type":"event","event":{"type":"chainsync.rollback","timestamp":"2024-10-14T15:03:49.820061849Z","payload":{"blockHash":"34385f4f0a8625df1025e6711057b688583266e7527b5a7d06550546f720473a","slotNumber":135791227}}}
```
although it complains about the casing of the address, using numeric ips doesn't help.
here's the relevant part of the config file:
```yml
# Plugin options
plugins:

  # Input plugin options
  input:
    # Chainsync input plugin options
    chainsync:
      network: mainnet
      address: cardano-node-mainnet:3001
      intersect-tip: false
      intersect-point: 135791219.79a9e7f9fd6d7b003f28e147c5d63b78c932e3ba3eb520c9f7e6fb16612c5d5a
      kupo-url: http://kupo:1442/
```
Additionally, although probably unrelated, the input resolution spams the output with these messages, at all loglevels:
```log
2024/10/14 15:03:49 finding matches: url=http://kupo:1442/v1/matches/2@c07d1d6206ffa022647392192583c85986b0b7927420f61770139a2002e035b2
2024/10/14 15:03:49 Matches() finished: duration=1ms matched=1 err=
2024/10/14 15:03:49 finding matches: url=http://kupo:1442/v1/matches/1@1850b309e74b8cd89cad02449545d0d403f185ef427e0b96ff6b58f5103a09d6
2024/10/14 15:03:49 Matches() finished: duration=1ms matched=1 err=
```
this is how I start adder:
```bash
docker run -it --rm --name adder -v ./adder.yml:/config.yml --network=ogmios_default -p 8080:8080  ghcr.io/blinklabs-io/adder -config /config.yml
```
let me know if you need further details! 
PS: In the meantime, the same config works by just reverting to 0.23.4 :shrug:

**Comment by verbotenj** [comment]
Fix in #261

**Comment by Javieracost** [comment]
confirmed working on 0.24.1! :clap:
can something be done about the flood of
```
2024/10/15 02:01:38 Matches() finished: duration=4ms matched=1 err=
2024/10/15 02:01:38 finding matches: url=http://kupo:1442/v1/matches/5@3e04c98469c4722ab3f1087b20081f6e60c0d4df7d249bc8297e55932b173c32
2024/10/15 02:01:38 Matches() finished: duration=4ms matched=1 err=
```
messages though? at a rate of hundreds per second it totally wrecks my terminal output :sweat:
